cmd/shipperctl/util/cmd_util.go

```diff
@@ -0,0 +1,34 @@
+package util
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+)
+
+// askForConfirmation asks the user for confirmation. A user must type in "y" or "n" and
+// then press enter. It has fuzzy matching, so "y", "Y" both count as
+// confirmations. If the input is not recognized, it will ask again. The function does not return
+// until it gets a valid response from the user.
+// taken from https://gist.github.com/r0l1/3dcbb0c8f6cfe9c66ab8008f55f8f28b
+func AskForConfirmation(stdin io.Reader, message string) (bool, error) {
```

Comment:
I had a few reasons to put it here. I didn't want to overcrowd decommission.go and I thought this might be useful in the future if we implement more shipperctl commands. Also, as it is not part of the `decommission cleanup` logic I wanted to put it elsewhere.
Not strongly opinionated about this though.

Reply:
Personally I like this in a separate package, it's definitely a method that could be reused by another command.

Reply:
About not overcrowding `decommission.go` that's true, it could always be in a separate file.

About packages. In go `packages` are the "reusable unit" we should care about, and  if I am not mistaken all the commands are in the same package for now, so having a separate package feels a bit strange, because this separate package you will create it's going to be used only by the `cmd` package.

Reply:
That's true
I want in the future to reformat shipperctl structure and be inspired by [github cli structure](https://github.com/cli/cli/tree/trunk/pkg/cmd/pr) more
I have reorganized the files in a way that maybe will make more sense now